examples/ITK_Example18_MONAI_affine_Elastix_nonlinear.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.    if USE_COMPILED:
</pre><p>Maybe add a comment here explaining when USE_COMPILED is true, and why it is treated differently </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKElastix/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>